cmd/lncli/cmd_pay.go

```diff
@@ -672,3 +855,14 @@ func sendToRouteRequest(ctx *cli.Context, req *lnrpc.SendToRouteRequest) error {
 
 	return nil
 }
+
+// ESC is the ASCII code for escape character
+const ESC = 27
+
+var clear = fmt.Sprintf("%c[%dA%c[2K", ESC, 1, ESC)
```

Comment:
Wouldn't something like `"\033[2K"` work?

Reply:
Also please add a comment what this escape code does.

Reply:
Comment added, but it was explained already right below.

I don't know if other sequences work. Is there an advantage to using something else? I am no expert on this.